### PR TITLE
Check if a item exist in stack or not

### PR DESCRIPTION
**Pull request by Akashpycs**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: data_structures/stacks/stack.py [hunk 1]
```diff
@@ -46,7 +46,11 @@ def is_empty(self):
     def size(self):
         """ Return the size of the stack."""
         return len(self.stack)
-
+    
+    def is_exist(self, item):
```
Comment (cclauss) [inline data_structures/stacks/stack.py:50]: Please implement https://docs.python.org/3/reference/datamodel.html#object.__contains__ instead.

File: data_structures/stacks/stack.py [hunk 2]
```diff
@@ -66,3 +70,5 @@ class StackOverflowError(BaseException):
     print("After push(100), the stack is now: " + str(stack))
     print("is_empty(): " + str(stack.is_empty()))
     print("size(): " + str(stack.size()))
+    num = 5
+    if stack.is_exist(num): print("num is in stack")
```
Comment (cclauss) [inline data_structures/stacks/stack.py:74]:
```suggestion
    num = 5
    if num in stack:
        print(f"{num} is in stack")
```

File: data_structures/stacks/stack.py [hunk 3]
```diff
@@ -46,7 +46,11 @@ def is_empty(self):
     def size(self):
         """ Return the size of the stack."""
         return len(self.stack)
-
+    
+    def __contains__(self, item):
+        """Check if a particular item is in stack"""
+        return item in self.stack
+
```
Comment (cclauss) [inline data_structures/stacks/stack.py:53]:
```suggestion

    def __contains__(self, item) -> bool:
        """Check if item is in stack"""
        return item in self.stack

```

**Comment by Akashpycs** [comment]
Done

**Pull request merged**